rel-1.0.0/README.md

```diff
@@ -75,14 +72,8 @@ This image contains the operating system with all of the native dependencies nee
 
 ## Windows Containers
 
-  Windows Containers images use the `microsoft/windowsservercore` and `microsoft/nanoserver` base OS images from Windows Server 2016 Technical Preview 5.  For more information on Windows Containers and a getting started guide, please see: [Windows Containers Documentation](http://aka.ms/windowscontainers).
+  Windows Containers images use the `microsoft/windowsservercore` base OS image from Windows Server 2016.  For more information on Windows Containers and a getting started guide, please see: [Windows Containers Documentation](http://aka.ms/windowscontainers).
 
--       `rel-1.0.0-windowsservercore-sdk`
 -       `rel-1.0.0-nanoserver-sdk`
--       `rel-1.0.0-windowsservercore-onbuild`
 -       `rel-1.0.0-nanoserver-onbuild`
--       `rel-1.0.0-windowsservercore-core`
 -       `rel-1.0.0-nanoserver-core`
-
-
-[dotnet/cli]: https://github.com/dotnet/cli
```

Comment:
This alias is still used in the introduction of this README.  I think it would make more sense to remove it when https://github.com/dotnet/dotnet-docker/pull/119 is ported to this repo.